### DISCUSSION TITLE
My small ideas 🪩

### DISCUSSION THREAD

**Discussion by BaptisteAdeliom** [agence-adeliom/adeliom-webcomponents#7] [Ideas]
- Add a component to load and display 3D objects (glb, gtlf, json etc...) in a basic scene.
- Add a component to display a map (refactor adeliom map js) and to work with all map suppliers.
- Add a component to add filters & pagination (refactor adeliom livesearch js).
- Add a component to add social network feed.

**Comment by BaptisteAdeliom** [comment]
- Add a component to load and display Spline files (https://www.npmjs.com/package/@splinetool/runtime)

**Comment by arnaud-ritti** [comment]
Map components IS done
